_posts/en/2021-12-8-throwback-to-nantes-2021-devfest-my-first-conference.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: Throwback to Nantes' 2021 DevFest, my first conference
+excerpt: "Finally! After 4 years of working as a web developer, I finally attended a conference linked to my job & passion: the DevFest Nantes."
+lang: en
+authors:
+    - jbberthet
+permalink: /throwback-to-nantes-2021-devfest-my-first-conference/
+categories:
+    - conference
+---
+
+![DevFest Nantes 2021's logo]({{ site.baseurl }}/assets/2021-12-08-mon-retour-sur-le-devfest-nantes-ma-premiere-conference/devfest-logo.png)
+
+Finally! After 4 years of working as a web developer, I finally attended a conference linked to my job & passion: the [DevFest Nantes](https://devfest2021.gdgnantes.com).
+
+For the 9th edition, which took place on 21 & 22 of October, the theme was street art. Both the Google Developer Group (GDG) of Nantes and his partners on the event did a great job and the final result was stunning ! Graffiti everywhere, basketball hoops, beers, video games and a photo booth were styling the lounge all along the event.
```

Comment:
```suggestion
For the 9th edition, which took place on 21 & 22 of October, the theme was street art. Both the Google Developer Group (GDG) of Nantes and his partners on the event did a great job and the final result was stunning! Graffiti everywhere, basketball hoops, beers, video games, and a photo booth were styling the lounge all along with the event.
```

Reply:
Here, I'll remove the space before the "!", nice catch! 👍  
Though I will not add the comma because of the "and" keyword doing the job. Finally, I don't see why you want to add "with" here, it sounds like you want to say "tout au long avec l'événement" in french which does not means much to me 🙂